homepage/homepage/content/docs/reference/testing.mdx

```diff
@@ -0,0 +1,214 @@
+import { Alert, CodeGroup, ContentByFramework } from '@/components/forMdx';
+
+export const metadata = {
+  description: "Guidance on testing Jazz apps."
+};
+
+# Testing Jazz Apps
+
+As you develop your Jazz app, you might find yourself needing to test functionality relating to sync, identities, and offline behaviour. The `jazz-tools/testing` utilities provide helpers to enable you to do so.
+
+## Core test helpers
+Jazz provides some key helpers that you can use to simplify writing complex tests for your app's functionality.
+
+### `setupJazzTestSync`
+
+This should normally be the first thing you call in your test setup, for example in a `beforeEach` or `beforeAll` block. This function creates and returns a plain `Account`, which will also act as an in-memory sync node for the test session. You'll often pair this with `createJazzTestAccount()` for more complex testing. Test data is not persisted, and no clean-up is needed between test runs. 
```

Comment:
This is actually useful only for testing multi-account interactions, but it's good to have this as a rule to avoid wasting time trying to understand why two accounts don't sync